File: fontes/br.com.wdc.shopping/br.com.wdc.shopping.view.remote/remote.shell.react/src/scripts/views/product/product-sel.ts
/**
 * Seletores CSS do módulo de produto (detalhe).
 * Importado por ProductView.tsx — não referenciar product.scss diretamente nas views.
 */
import GlobalSel from "@root/global-sel"

import "./product.scss"

const ProductSel = {
  ...GlobalSel,

  // :: Product detail

  title: "wdc-product__title",
  divider: "wdc-product__divider",
  descCard: "wdc-product__desc-card",
  descText: "wdc-product__desc-text",
  priceImageRow: "wdc-product__price-image-row",
  priceCol: "wdc-product__price-col",
  priceBadge: "wdc-product__price-badge",
  qtyRow: "wdc-product__qty-row",
  qtyLabel: "wdc-product__qty-label",
  qtyValue: "wdc-product__qty-value",
  imageBox: "wdc-product__image-box",
  image: "wdc-product__image",
  actionsRow: "wdc-product__actions-row",
} as const

export default ProductSel

File: fontes/br.com.wdc.shopping/br.com.wdc.shopping.view.remote/remote.shell.codenameone/common/src/main/java/br/com/wdc/shopping/view/remote/shell/cn1/views/product/ProductSel.java
package br.com.wdc.shopping.view.remote.shell.cn1.views.product;

import br.com.wdc.shopping.view.remote.shell.cn1.Sel;

/** Seletores (UIIDs) do detalhe do produto — espelha {@code views/product/_product.scss}. Herda os globais. */
public final class ProductSel extends Sel {

    public static final ProductSel INSTANCE = new ProductSel();

    private ProductSel() {
        // singleton
    }

    public final String PRODUCT_PAGE = "ProductPage";
    public final String PRODUCT_TITLE = "ProductTitle";
    public final String PRODUCT_DIVIDER = "ProductDivider";
    public final String PRODUCT_DESC_CARD = "ProductDescCard";
    public final String PRODUCT_DESC_TEXT = "ProductDescText";
    public final String PRODUCT_PRICE_IMAGE_ROW = "ProductPriceImageRow";
    public final String PRODUCT_PRICE_BADGE = "ProductPriceBadge";
    public final String QTY_LABEL = "QtyLabel";
    public final String PRODUCT_IMAGE_BOX = "ProductImageBox";
}
